pkg/tsdb/mssql/mssql.go

```diff
@@ -52,12 +52,14 @@ func generateConnectionString(datasource *models.DataSource) string {
 	}
 
 	server, port := hostParts[0], hostParts[1]
-	return fmt.Sprintf("server=%s;port=%s;database=%s;user id=%s;password=%s;",
+	encrypt := datasource.JsonData.Get("encrypt").MustString("false")
+	return fmt.Sprintf("server=%s;port=%s;database=%s;user id=%s;password=%s;encrypt=%s;",
```

Comment:
After testing this seems like some other settings changes as soon as you specify the `encrypt` setting, see `TrustServerCertificate` [here](https://github.com/denisenkom/go-mssqldb#less-common-parameters). Didn't know that up until know.

Because of this I think the easiest thing would be to only populate connectionstring with `encrypt` if value is not `false` since that is the third party library default. The alternative would be to set the `TrustServerCertificate` as well, but feels better to let the third party lib care about this specific case.